readinglistmanager/db.py

```diff
@@ -231,16 +231,17 @@ def getListNames(self):
         printResults("Getting list names for source: %s" %
                      (self.source.name), 3)
         if isinstance(self.source, OnlineSource):
+            readingListTitles = []
             cursor = self.connection.cursor()
 
             try:
-                listQuery = ''' SELECT * FROM %s ''' % (
+                listQuery = ''' SELECT olistnum, name FROM %s ''' % (
                     self.source.tableReadingListTitles)
                 readingListTitles = cursor.execute(listQuery).fetchall()
             except Exception as e:
-                printResults("Warning : Unable to find DB table details for %s" % (
-                    self.source.name), 5)
-                print(e)
+                print("Caught error:", e)
```

Comment:
The printResults function includes printing to console with print(), so this is redundant when coupled with the following line.